pyccel/ast/core.py

```diff
@@ -195,6 +195,8 @@
     'Point': Symbol('Point')
 }
 
+#==============================================================================
+def apply(func, args, kwargs):return func(*args, **kwargs)
```

Comment:
This function needs a docstring...

Maybe what it does is clear, but it is definitely not clear why it is needed and what it is used for.

Reply:
it's used by pickle's methods that I have added, you can find the doc strings there